src/selectors/entities/channels.js

```diff
@@ -428,24 +431,42 @@ export const getUnreads = createSelector(
                 if (channel.type === 'D') {
                     otherUserId = getUserIdFromChannelName(currentUserId, channel.name);
                     if (users[otherUserId] && users[otherUserId].delete_at === 0) {
```

Comment:
We do a lot of calc here in this selector to get the values of count based on archived channels and deleted channel. So far it works fine because we have the information of the archived channels for the current team.

Now here with this change we are trying to add count from all teams which might have count from archived channels which can pose an issue with unread count being out of sync. This probably will be considered edge case but i want to point that out as to think of a better solution around this.